includes/pipeline_page/sidebar.php

```diff
@@ -46,9 +52,13 @@
 ?>
 
 <div class="pipeline-sidebar">
-  <h6><i class="fas fa-terminal fa-xs"></i> command</h6>
-  <div class="border pipeline-run-cmd p-1">
-    <code class="small"> nextflow run <?php echo $pipeline->full_name; echo $release_cmd; ?> -profile test</code>
+  <div class="row border-bottom pb-2">
+    <div class="col-12">
```

Comment:
What's the advantage of adding these divs?

Reply:
row elements are somehow wider than non row elements (see bottom border of collaborators field in the old version)

Reply:
Ah right, yeah this is a mistake that I make a lot. I think it's better to move the `bottom-border` class off the `.row` element rather than making everything rows. The `.row` has a negative margin to cope with the padding on `.col` elements.

Reply:
But I like the overhanging borders and wouldn't like to sacrifice it for padding inwards (because that would probably be again to narrow for the command etc.)

Reply:
hah, ok ¯\_(ツ)_/¯

Reply:
NB: I never intended the side bar to be wide enough to accommodate the command. What I meant to do (and never got around to) was to add a copy-button like the one that GitHub has for cloning repos:

<img src="https://user-images.githubusercontent.com/465550/88583149-21211a00-d050-11ea-8595-8d421d780fb2.png" width=400>

Reply:
Good, because I also want to have this copy button. I'll add it then. I still like the full width command because it makes it clearer what one is copying and get familiar with the nextflow command syntax.